src/Core/tests/DeviceTests.Shared/Stubs/ContextStub.cs

```diff
@@ -38,6 +38,12 @@ public object GetService(Type serviceType)
 
 			if (serviceType == typeof(NavigationRootManager))
 				return _windowManager ??= new NavigationRootManager(this);
+
+			if (serviceType == typeof(IFontManager))
```

Comment:
@jsuarezruiz I think this change also is what broke the API 23 tests 

If I remove this code then the `FontFamilyInitializesCorrectly` tests all pass correctly.